python/lsst/ts/m2gui/controltab/tab_settings.py

```diff
@@ -120,24 +155,40 @@ def _set_minimum_width_line_edit(self, line_edit, offset=20):
         line_edit.setMinimumWidth(width + offset)
 
     @asyncSlot()
-    async def _callback_apply(self):
-        """Callback of the apply button. This will apply the settings."""
+    async def _callback_apply_host(self):
+        """Callback of the apply-host-setting button. This will apply the
+        new host settings to model."""
 
         host = self._settings["host"].text()
         port_command = self._settings["port_command"].value()
         port_telemetry = self._settings["port_telemetry"].value()
         timeout_connection = self._settings["timeout_connection"].value()
 
-        is_successful = await run_command(
+        await run_command(
             self.model.update_connection_information,
             host,
             port_command,
             port_telemetry,
             timeout_connection,
         )
 
-        if is_successful:
-            self.model.log.setLevel(self._settings["log_level"].value())
+    @asyncSlot()
+    async def _callback_apply_general(self):
+        """Callback of the apply-general-setting button. This will apply the
+        new general settings to model."""
+
+        self.model.log.setLevel(self._settings["log_level"].value())
+
+        # The unit of self.model.duration_refresh is milliseconds
+        self.model.duration_refresh = int(
+            1000 / self._settings["refresh_frequency"].value()
+        )
+
+        # Update the point size
```

Comment:
Display point size should be managed by the hosting window manager, not in-application. Yes, this gives a benefit when running on Mac in Docker, but it's better to fix the root case (there should be an argument for point size substitutions e.g, in XQuartz)

Reply:
I tried to provide an easy way for the individual user to modify the size of font. I thought it might not be easy for the main users (hardware engineers or operators) to learn the idea of `XQuartz`. In addition, the EUI will be deployed by `puppet` and maybe the user does not have the enough authority to modify the settings of system except the EUI itself in the runtime (at least, my authority is very minimum for the puppetized machines on summit). Thanks!